docs/samples/contact-center/app.js

```diff
@@ -42,6 +31,11 @@ const incomingDetailsElm = document.querySelector('#incoming-call');
 const answerElm = document.querySelector('#answer');
 const declineElm = document.querySelector('#decline');
 const callControlListener = document.querySelector('#callcontrolsection');
+const holdElm = document.querySelector('#hold');
+const resumeElm = document.querySelector('#resume');
+const endElm = document.querySelector('#end');
+const wrapupElm = document.querySelector('#wrapup');
+const wrapupCodesDropdownElm = document.querySelector('#wrapupCodesDropdown');
```

Comment:
We will now store the entire list of codes using this dropdown and populate the request accordingly.